yarn.lock

```diff
@@ -4,27 +4,27 @@
 
 "@ampproject/remapping@^2.2.0":
   version "2.2.0"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz"
```

Comment:
Is there a reason why this PR changes all these packages to resolve to npmjs.org instead of yarnpkg.com?

Reply:
because I accidentally used npm instead of yarn! good catch, will fix

Reply:
resolved